Size of `TransformedShape` and `Body` is not as expected on ARM32 with double-precision

When compiling with `DOUBLE_PRECISION=YES` for ARM32 Android, I'm met with the following errors:

```
Jolt/Physics/Collision/TransformedShape.h:191:1: error: static_assert failed due to requirement 'sizeof(JPH::TransformedShape) == 96' "Not properly packed"
static_assert(sizeof(TransformedShape) == JPH_IF_SINGLE_PRECISION_ELSE(64, 96), "Not properly packed");
^             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
```
Jolt/Physics/Body/Body.h:385:1: error: static_assert failed due to requirement 'sizeof(JPH::Body) == 160' "Body size is incorrect"
static_assert(sizeof(Body) == JPH_IF_SINGLE_PRECISION_ELSE(128, 160), "Body size is incorrect");
^             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

I've only looked into `TransformedShape` so far, but I assume the issue is similar with `Body`.

Basically, it seems like this:

https://github.com/jrouwe/JoltPhysics/blob/4615f9ca7fc15e3a114986a5c7339933be56e8a1/Jolt/Core/Core.h#L175-L176

... coupled with the fact that ARM32 uses 32-bit pointers, leads to `TransformedShape` having a total size of 80 bytes (including a 4-byte padding at the end), which obviously differs from the 96 bytes expected in this assertion:

https://github.com/jrouwe/JoltPhysics/blob/4615f9ca7fc15e3a114986a5c7339933be56e8a1/Jolt/Physics/Collision/TransformedShape.h#L191

--- a/Jolt/Physics/Body/Body.h
+++ b/Jolt/Physics/Body/Body.h
@@ -375,14 +375,9 @@ private:
 	atomic<uint8>			mFlags = 0;														///< See EFlags for possible flags
 
 	// 122 bytes up to here (64-bit mode, single precision, 16-bit ObjectLayer)
-
-#if JPH_CPU_ADDRESS_BITS == 32
-	// Padding for mShape, mMotionProperties, mCollisionGroup.mGroupFilter being 4 instead of 8 bytes in 32 bit mode
-	uint8					mPadding[12];
-#endif
 };
 
-static_assert(sizeof(Body) == JPH_IF_SINGLE_PRECISION_ELSE(128, 160), "Body size is incorrect");
+static_assert(JPH_CPU_ADDRESS_BITS != 64 || sizeof(Body) == JPH_IF_SINGLE_PRECISION_ELSE(128, 160), "Body size is incorrect");
 static_assert(alignof(Body) == JPH_RVECTOR_ALIGNMENT, "Body should properly align");
 
 JPH_NAMESPACE_END

--- a/Jolt/Physics/Collision/TransformedShape.h
+++ b/Jolt/Physics/Collision/TransformedShape.h
@@ -188,7 +188,7 @@ public:
 	SubShapeIDCreator			mSubShapeIDCreator;							///< Optional sub shape ID creator for the shape (can be used when expanding compound shapes into multiple transformed shapes)
 };
 
-static_assert(sizeof(TransformedShape) == JPH_IF_SINGLE_PRECISION_ELSE(64, 96), "Not properly packed");
+static_assert(JPH_CPU_ADDRESS_BITS != 64 || sizeof(TransformedShape) == JPH_IF_SINGLE_PRECISION_ELSE(64, 96), "Not properly packed");
 static_assert(alignof(TransformedShape) == JPH_RVECTOR_ALIGNMENT, "Not properly aligned");
 
 JPH_NAMESPACE_END

